examples/cube.rs

```diff
@@ -112,30 +118,49 @@ pub fn setup_mpm_particles(
         dt: (1.0 / 60.0) / (app_state.num_substeps as f32),
     };
 
-    let cell_width = 0.5;
+    let cell_width = 1.0;
     let mut particles = vec![];
 
-    for rock in &rocks {
-        let position = vector![rock.x, rock.y, rock.z];
-
-        let rock_size = vector![1.0, 1.0, 1.0];
-        let volume = rock_size.x * rock_size.y * rock_size.z;
-        let density = 2700.0;
-        particles.push(Particle {
-            position: vector![position.x, position.y, position.z],
-            velocity: Vector3::zeros(),
-            volume: ParticleMassProps::new(density * volume, volume.cbrt() / 2.0),
-            model: ElasticCoefficients::from_young_modulus(10_000_000.0, 0.2),
-            plasticity: Some(DruckerPrager {
-                // TODO: tune these values.
-                h0: 45.0f32.to_radians(),
-                h1: 50.0f32.to_radians(),
-                h2: 0.4,
-                h3: 15.0f32.to_radians(),
-                ..DruckerPrager::new(10_000_000.0, 0.2)
-            }),
-            phase: None,
-        });
+    for x in -1..2 {
+        for z in -1..2 {
+            let x = x as f32;
+            let z = z as f32;
+            let offset = vector![x * grid_size_x as f32, 0f32, z * grid_size_z as f32] * 2f32;
+            for particle in &particle_positions {
+                let position = vector![particle.x, particle.y, particle.z];
+
```

Comment:
those are way outside the scope of this PR, and lacking proper polish, but I'll roll with it as this project is still heavily WIP.